.github/workflows/gradle-build-feature.yml

```diff
@@ -27,6 +41,23 @@ jobs:
         run: chmod +x gradlew
       - name: Gradle package
         run: ./gradlew assemble
+      - name: Build docker image
+        run: ./gradlew dockerBuild
+      - name: Run docker container
+        run: ./gradlew dockerRun
+      - name: Cypress runs tests
+        uses: cypress-io/github-action@v2
+        with:
+          working-directory: web-ui
+          config: pageLoadTimeout=100000,baseUrl=http://localhost:8080
+          build: yarn build
+          start: yarn start
```

Comment:
if we are building and running a docker image above (and pointing to that running image), why do we need the yarn build and yarn start here?

Reply:
It works without it. I deleted it. Thanks!